src/FrameMessageTypes.d.ts

```diff
@@ -46,7 +46,7 @@ export interface InitApiSdkResponse {
 }
 export interface CreateUserRequest {
     type: "CREATE_USER";
-    message: {passcode: string; jwtToken: string};
+    message: {passcode: string; jwtToken: string; needsRotation?: boolean};
```

Comment:
For this message, the `needsRotation` boolean should be set so you should remove the `?`.